### ISSUE TITLE
Spacing of Network Settings is not consistent

### ISSUE THREAD

**Issue by Asgaror** [skycoin/skyflash#29]
*Assume minions will be renamed to nodes, related to* #26 

**Is your feature request related to a problem? Please describe.**
The network settings that are being displayed in the GUI have a strange & inconsistent spacing format: 
![image](https://user-images.githubusercontent.com/32180146/59097268-6bf84380-891d-11e9-92be-6dd018033dd6.png)

**Describe the solution you'd like**
Consistent formatting like:
```
Network gateway: 192.168.0.1
Network DNS: 1.0.0.1, 1.1.1.1
Manager IP: 192.168.0.2
Nodes count: 7 
```

**Comment by stdevPavelmc** [comment]
The spacing of the IPs in the UI is due to the QML filtering for IPs, in that way we do the filtering on the UI layer itself, so the user can't enter a IP like 345.678.456.986, a side effect is the spacing you see.

Can be changed to more consistent spacing but would require more code under the hood...

**Comment by stdevPavelmc** [comment]
Closed via PR #47
